docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -274,5 +482,6 @@ ALTER TABLE dbx.tab1 UNSET TBLPROPERTIES ('winner')
 ### Related Statements
 - [CREATE TABLE](sql-ref-syntax-ddl-create-table.html)
 - [DROP TABLE](sql-ref-syntax-ddl-drop-table.html)
+- [ALTER TABLE ... ADD PARTITION](sql-ref-syntax-ddl-alter-table.html)
```

Comment:
is this change intentional?

Reply:
oh, I thought that one is part of umbrella jira, so tried to reference. Thanks for catching this, I will remove it.